front/app/containers/App/index.tsx

```diff
@@ -390,7 +390,6 @@ const App = ({ children }: Props) => {
                 flexDirection="column"
                 alignItems="stretch"
                 flex="1"
-                overflowY="auto"
                 id="main-content"
```

Comment:
In order for `position: sticky` to work in child elements, there can't be an `overflowY="auto"` set anywhere in parent elements.

After removing this from the App file, I looked all over the platform but didn't see any strange UI, so I don't think it's actually needed here :thinking: I'll set up an epic deployment and maybe you could do a quick check as well though? :pray:

Reply:
Future reference: that line was coming from [this PR](https://github.com/CitizenLabDotCo/citizenlab/pull/5981).